doc/user/content/sql/create-sink.md

```diff
@@ -54,7 +54,8 @@ Field                | Value type | Description
 ---------------------|------------|------------
 `partition_count`    | `int`      | Set the sink Kafka topic's partition count. This defaults to -1 (use the broker default).
 `replication_factor` | `int`      | Set the sink Kafka topic's replication factor. This defaults to -1 (use the broker default).
-`consistency_topic`  | `text`     | Makes the sink emit additional [consistency metadata](#consistency-metadata). Only valid for Kafka sinks. This defaults to empty, meaning no consistency information is emitted by default.
+`exactly_once`  | `bool`   |  Use the existing Kafka topic after Materialize restarts, instead of creating a new one. `consistency_topic` is required if true. The default is false.
```

Comment:
Right now, this describes what it is used for, not what it's actual meaning is. I don't know which is better, to be honest, but an correct description of this flag would be:

> Ensure that each input change/update is only reflected once in the output topic. Simply speaking, this ensures that no duplicates are written to the output topic.

In the section about re-using the output topic name, we could then describe that enabling `exactly-once` is required to achieve it? What do you think?

cc @ruchirK

Reply:
I don't have a strong preference. I think that from a user's perspective they won't really be thinking about "does Materialize write duplicate data" because they will assume that it doesn't by default? For me, the user facing effect that this option has is that when its set to true, Materialize reuse the existing sink topic and without it Materialize creates a new topic each startup. Perhaps that means the option should be renamed to "reuse-topic" or something?

Regardless, after typing this out I don't think we should mention duplicates in the user facing docs because that feels like implementation details.

I don't feel very confident in my assessment here and am happy to defer to whatever you both think!

Reply:
I think "exactly-once" and the details of duplication are the kind of internal things users don't need details on; this is basically making sinks act the way you'd *expect* them to act. At least, as a nontechnical user, the whole thing about creating a new topic after restart is unintuitive.

I like the idea of renaming the option "reuse-topic". "topic-persistence" is a possibility, but seems too easy to confuse with other types of persistence.

Reply:
Oh boy, as always, naming is the hard thing. I would defer to Micole, or we could ask product, or the source/sink team.

Reply:
@ruchirK - Can we go forward with the renaming you suggested?

Reply:
Yes, please go ahead. The PR to change the setting will be merged once it's green: https://github.com/MaterializeInc/materialize/pull/7492